src/OpenLoco/GameCommands/GameCommands.h

```diff
@@ -1577,25 +1578,33 @@ namespace OpenLoco::GameCommands
         doCommand(GameCommand::multiplayerSave, regs);
     }
 
-    // Update owner status
-    inline void do_73(EntityId id)
+    struct UpdateOwnerStatusArgs
     {
-        registers regs;
-        regs.bl = Flags::apply;
-        regs.ax = -2;
-        regs.cx = enumValue(id);
-        doCommand(GameCommand::updateOwnerStatus, regs);
-    }
+        static constexpr auto command = GameCommand::updateOwnerStatus;
+        UpdateOwnerStatusArgs() = default;
+        explicit UpdateOwnerStatusArgs(const registers& regs)
+        {
+        }
 
-    // Update owner status
-    inline void do_73(Map::Pos2 position)
-    {
-        registers regs;
-        regs.bl = Flags::apply;
-        regs.ax = position.x;
-        regs.cx = position.y;
-        doCommand(GameCommand::updateOwnerStatus, regs);
-    }
+        OwnerStatus ownerStatus;
+
+        explicit operator registers() const
+        {
+            registers regs;
+            if (ownerStatus.isEntity())
```

Comment:
you dont need to do this switch this is already handled by the constructor of the class. Just provide some sort of way to get the data.